Src/GoogleApis.Auth.DotNet4/OAuth2/GoogleWebAuthorizationBroker.cs

```diff
@@ -113,17 +113,22 @@ public class GoogleWebAuthorizationBroker
         /// <param name="taskCancellationToken">Cancellation token to cancel an operation.</param>
         /// <param name="dataStore">The data store, if not specified a file data store will be used.</param>
         /// <returns>User credential.</returns>
-        private static async Task<UserCredential> AuthorizeAsyncCore(
+        public static async Task<UserCredential> AuthorizeAsyncCore(
             GoogleAuthorizationCodeFlow.Initializer initializer, IEnumerable<string> scopes, string user,
-            CancellationToken taskCancellationToken, IDataStore dataStore = null)
+            CancellationToken taskCancellationToken, IDataStore dataStore = null,
+            bool? includeGrantedScopes = null)

```

Comment:
Makes sense, I was attempting to stick with the existing setup. For instance, scopes and datastore, which are also part of the initializer, are included. I wasn't sure if that was intentional or for legacy code so I erred on the side of intentional. I agree that it'd be cleaner to not have it there, and will remove it.